### ISSUE TITLE
Font Property?

### ISSUE THREAD

**Issue by theryan722** [jacobslusser/ScintillaNET#124]
In 2.0 there was a Font property. There isn't in this one. Is this possible to add?

**Comment by jonandr** [comment]
Font is available on styles, you can see how it is used in the wiki:

https://github.com/jacobslusser/ScintillaNET/wiki/Automatic-Syntax-Highlighting

If you're not using a lexer you can just change the font for the ScintillaNET.Style.Default style.

**Comment by theryan722** [comment]
I know that. However it doesn't always react well with storing it as a font variable and using a font dialog.

**Comment by jacobslusser** [comment]
The absence of a Font property of type `System.Drawing.Font` is intentional. In v2 it actually didn't work well/correctly because that object cannot roundtrip from managed code to native and back again. Thus we had to fake it and under the covers it was messy. 

Like many of the other changes in v3 I chose to follow the native Scintilla API more closely and thus use only the font name as a `String` the way the native Scintilla control does.
